operator/controller/src/main/java/io/apicurio/registry/operator/resource/ResourceFactory.java

```diff
@@ -49,11 +50,17 @@ public Deployment getDefaultAppDeployment(ApicurioRegistry3 primary) {
         } else {
             r.getSpec().setTemplate(new PodTemplateSpec());
         }
+
+        var imageOpt = ConfigProvider.getConfig().getOptionalValue("registry.app.image", String.class);
```

Comment:
let's check if we can use `@ConfigProperty`

Reply:
even if we want to keep this as a future task, please, at least group the configuration lookups in separate methods instead of doing it inline in a bigger piece of logic.

Reply:
Done.